src/components/SearchBar/_search-bar.scss

```diff
@@ -0,0 +1,34 @@
+.search-container {
+  display: flex;
+  flex-flow: row nowrap;
+}
+
+.search-input {
+  height: 40px;
+  width: 60%;
+  border-radius: 10px;
+  font-size: large;
+  font-weight: 700;
+  font-family: $inconsolata;
+  padding-left: 1vw;
+  background-color: $mid-emphasis-text;
+
+  &:focus {
+    background-color: whitesmoke;
+  }
+}
+
+.search-btn {
+  height: 45px;
+  width: 10vw;
+  margin: 0 0 0 20px;
+  background-color: $blue-jeans;
+  border-radius: 10px;
+  font-size: large;
+  font-weight: 600;
+  font-family: $inconsolata;
+
```

Comment:
I'm going to try to amalgamate our hover/effects and changes into one fantastical search button